The Rust compiler does not implement a fallback for unknown target triple
vendors (e.g. "cros" or "pc") to the "unknown" variant, so we add copies of the
relevant cros targets here as well.

--- a/src/librustc_target/spec/mod.rs
+++ b/src/librustc_target/spec/mod.rs
@@ -492,6 +492,7 @@ macro_rules! supported_targets {
 supported_targets! {
     ("x86_64-unknown-linux-gnu", x86_64_unknown_linux_gnu),
     ("x86_64-unknown-linux-gnux32", x86_64_unknown_linux_gnux32),
+    ("x86_64-pc-linux-gnu", x86_64_pc_linux_gnu),
     ("i686-unknown-linux-gnu", i686_unknown_linux_gnu),
     ("i586-unknown-linux-gnu", i586_unknown_linux_gnu),
     ("mips-unknown-linux-gnu", mips_unknown_linux_gnu),
